### PR TITLE
feat: Blazor example

### LINKED ISSUE #89: Adding support to run in blazor wasm ?
I am trying to get the client db api to run in  a blazor app and am getting a platform not supported error, are you planning to allow the library to work with blazor web assembly ?

`    var influxDBClient = InfluxDBClientFactory.Create("https://us-west-2-1.aws.cloud2.influxdata.com/", Token);

        int i = 0;

        var fluxTables = await influxDBClient.GetQueryApi().QueryAsync(flux, orgid);
        fluxTables.ForEach(fluxTable =>
        {
            var fluxRecords = fluxTable.Records;
            fluxRecords.ForEach(fluxRecord =>
            {
                Console.WriteLine(i + "\t" + $"{fluxRecord.GetTime()}\t{fluxRecord.GetValue()}\t{fluxRecord.GetValueByIndex(8)}");
                i++;
            });
        });
`

### PR DESCRIPTION
**Pull request by michaelahojna**
Closes #89

## Proposed Changes

 Added example how to use C# Client in Blazor app

## Checklist

- [x] CHANGELOG.md updated
- [x] Rebased/mergeable
- [x] `dotnet test` completes successfully
- [x] Commit messages are [conventional](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ExampleBlazor/ExampleBlazor.csproj [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="InfluxDB.Client" Version="4.4.0-dev.6843" />
```
Comment (bednar) [inline ExampleBlazor/ExampleBlazor.csproj:10]: The package should be referenced via relative path.

```suggestion
      <ProjectReference Include="..\Client\Client.csproj" />
```

**Review by bednar — CHANGES_REQUESTED** [review]

**Review by bednar — CHANGES_REQUESTED** [review]
@michaelahojna, thanks for your PR 👍 

Please reformat sources to required style by: `./Scripts/reformat-code.sh` and create `README.md` with description of the **Blazor example**.

**Review by bednar — CHANGES_REQUESTED** [review]
Please rebase sources and update `CHANGELOG.md` and also PR description:

image

**Comment by rhajek** [comment]
The demo is missing an example how to use LINQ somewhere in `InfluxModel.cs`.

File: Examples/ExampleBlazor/Data/InfluxModel.cs [hunk 2]
```diff
@@ -0,0 +1,696 @@
...
+                .First().Id;
+        }
+        catch (Exception e)
+        {
+            Debug.WriteLine("Failed to load organization Id." + e);
+        }
+        finally
+        {
+            influxDbClient.Dispose();
+        }
```
Comment (bednar) [inline Examples/ExampleBlazor/Data/InfluxModel.cs:42]: Please use `using` keyword instead explicitly disposing client. Use this behaviour on all places.

```suggestion
        using var influxDbClient = client.GetClient();
        try
        {
            var orgList = await influxDbClient.GetOrganizationsApi().FindOrganizationsAsync(org: client.Org);

            var tmp = orgList.First();
            return (await influxDbClient.GetOrganizationsApi().FindOrganizationsAsync(org: client.Org))
                .First().Id;
        }
        catch (Exception e)
        {
            Debug.WriteLine("Failed to load organization Id." + e);
        }
```

File: Examples/ExampleBlazor/Data/ClientSettings.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    public InfluxDBClient GetClient(double timespanSeconds)
+    {
+        var options = new InfluxDBClientOptions.Builder()
+            .Url(Url)
+            .AuthenticateToken(Token)
+            .TimeOut(TimeSpan.FromSeconds(timespanSeconds))
+            .Build();
+
+        return InfluxDBClientFactory.Create(options);
+    }
```
Comment (bednar) [inline Examples/ExampleBlazor/Data/ClientSettings.cs:56]: Use the optional parameter to simplify code:

```suggestion
    public InfluxDBClient GetClient(double timespanSeconds = 10)
    {
        var options = new InfluxDBClientOptions.Builder()
            .Url(Url)
            .AuthenticateToken(Token)
            .TimeOut(TimeSpan.FromSeconds(timespanSeconds))
            .Build();

        return InfluxDBClientFactory.Create(options);
    }
```

**Review by bednar — CHANGES_REQUESTED** [review]

**Review by bednar — APPROVED** [review]
LGTM 🚀

**Pull request merged**